subprojects/configuration-cache/src/main/kotlin/org/gradle/configurationcache/initialization/ConfigurationCacheStartParameter.kt

```diff
@@ -136,6 +136,9 @@ class ConfigurationCacheStartParameter(
 
     val isNoBuildScan: Boolean
         get() = startParameter.isNoBuildScan
+
+    val isIsolatedProjects: Boolean
+        get() = modelParameters.isIsolatedProjects
```

Comment:
☁️  I intentionally used the `modelParameters` here to rely on the "more final" value of the Isolated Projects settings. The value in the `StartParameterInternal` is not final, because it can be affected by other build flags.

Reply:
💅 It makes sense to add a similar comment to the code.

Reply:
Done